Userland/Libraries/LibJS/Runtime/Value.h

```diff
@@ -398,6 +398,24 @@ class Value {
     template<typename... Args>
     [[nodiscard]] ALWAYS_INLINE ThrowCompletionOr<Value> invoke(VM&, PropertyKey const& property_key, Args... args);
 
+    static FlatPtr extract_pointer_bits(uint64_t encoded)
```

Comment:
`uint64_t`?

One of the call sites passes FlatPtr here, which is 32 bits sometimes (Heap.cpp).

At any rate it should be u64 since Value::m_bits::encoded is u64

Reply:
Side note can this be a constexpr method?

Reply:
This method shouldn't be called from Heap when when FlatPtr is 32 bits.
But yes it should be `u64` and can be constexpr